src/lib.rs

```diff
@@ -161,12 +196,20 @@ pub fn xtask_command() -> Command {
     Command::new(env::args_os().next().unwrap())
 }
 
-/// Watches over your project's source code, relaunching the given command when
+/// Watches over your project's source code, relaunching a given command when
 /// changes are detected.
 #[non_exhaustive]
 #[derive(Debug, Parser)]
+#[clap(
+    about = "Watches over your project's source code.",
+    long_about = "Watches over your project's source code. \n\
+        Re-launching a given command when changes are detected.
```

Comment:
The end user can't give that command but this help message is intended for the end user.